text-kit/autogen.kit.d/base.yml

```diff
@@ -6,6 +6,10 @@ dev_github:
     category: app-text
     github:
       query: tags
+    transform:
+        - kind: string
+          match: "v"
```

Comment:
This is done automatically from `mark-devkit` IIRC. What is your error without it?